shell/mixins/brand.js

```diff
@@ -144,6 +144,44 @@ export default {
       }
     }
   },
+  mounted() {
+    const cssClass = `overflow-hidden dashboard-body`;
+    const body = document.getElementsByTagName('body')[0];
+    let title = getVendor();
+    let bodyClass, brandMeta;
+
+    if (getVendor() === 'Harvester') {
+      const ico = require(`~shell/assets/images/pl/harvester.png`);
+
+      title = 'Harvester';
+      const link = document.createElement('link');
+
+      link.hid = 'icon';
+      link.rel = 'icon';
+      link.type = 'image/x-icon';
+      link.hrefv = ico;
+      const head = document.getElementsByTagName('head')[0];
+
+      head.appendChild(link);
+    }
+
+    if ( this.brand ) {
+      try {
+        brandMeta = require(`~shell/assets/brand/${ this.brand }/metadata.json`);
+        if (brandMeta?.hasStylesheet === 'true') {
+          bodyClass = `${ cssClass } ${ this.brand } theme-${ this.theme }`;
+        } else {
+          bodyClass = `theme-${ this.theme } overflow-hidden dashboard-body`;
+          this.$store.dispatch('prefs/setBrandStyle', this.theme === 'dark');
+        }
+      } catch {
+        bodyClass = `theme-${ this.theme } ${ cssClass }`;
+      }
+    }
+
+    document.title = title;
+    body.className = bodyClass;
```

Comment:
This will set `body.className` to `'undefined'` when `this.brand` is falsey - the default value for the brand setting is '' so by default the body class is set to`'undefined'`. The dashboard defaults to a light theme, so you'll have to switch to dark theme to observe that the theme isn't being applied.